### PR TITLE
125 update with edits sent to lsp for publication

### PR DESCRIPTION
**Pull request by amilan17**
#125

### CODE REVIEW

File: standard/requirements/core/REQ_data_id.adoc [hunk 1]
```diff
@@ -2,6 +2,7 @@
 [width="90%",cols="2,6a"]
 |===
 ^|*Requirement {counter:req-id}* |*/req/core/data_id*
-^|A |A WNM SHALL provide a `+properties.data_id+` property.
+^|A |A WNM `SHALL` provide a `+properties.data_id+` property.
```
Comment (amilan17) [inline standard/requirements/core/REQ_data_id.adoc:5]:
```suggestion
^|A |A WNM SHALL provide a `+properties.data_id+` property.
```

File: standard/requirements/core/REQ_validation.adoc [hunk 2]
```diff
@@ -3,7 +3,7 @@
 |===
 ^|*Requirement {counter:req-id}* |*/req/core/validation*
 ^|A |Each WNM SHALL validate without error against the WNM schema.
-^|B |Each WNM SHALL provide `+id+`, `+type+`, `+geometry+` and `+properties+` properties for GeoJSON compliance.
+^|B |Each WNM SHALL provide `+id+`, `+type+`, `+geometry+` and `+properties+` properties for GeoJSON compliance. (see <<WMO schemas server>>)
```
Comment (amilan17) [inline standard/requirements/core/REQ_validation.adoc:6]:
```suggestion
^|B |Each WNM SHALL provide `+id+`, `+type+`, `+geometry+` and `+properties+` properties for GeoJSON compliance.
```

File: standard/sections/clause_7_normative_text.adoc [hunk 3]
```diff
@@ -1,128 +1,119 @@
...
-|**required**
-|A fixed value denoting the record as a GeoJSON ``Feature`` (see <<_geojson_compliance>>)
+|**Required**
+|A fixed value denoting the record as a GeoJSON ``Feature`` (see <<_1_3_geojson_compliance>>)
 
 
 |``conformsTo``
-|**required** (one of ``conformsTo`` or ``version``)
-|The version of WNM associated that the record conforms to (see <<_conformance_2>>.
+|**Required** (one of ``conformsTo`` or ``version``)
```
Comment (amilan17) [inline standard/sections/clause_7_normative_text.adoc:28]: to discuss

File: standard/sections/clause_7_normative_text.adoc [hunk 4]
```diff
@@ -437,11 +432,11 @@ include::../requirements/core/REQ_links.adoc[]
 include::../recommendations/core/REC_links.adoc[]
 include::../recommendations/core/PER_links.adoc[]
 
-===== Access control
+===== 1.16.1 Access control
 
-For recommended data, WNM links may also provide links to resources that implement access control in support of authentication and authorization. In secure data use cases, a user needs to be able to detect access controlled data as part of data discovery and evaluation. The example demonstrates how to express access control using HTTP Basic Authentication for a given data access service.
+For recommended data, WNM links may also provide links to resources that implement access control in support of authentication and authorization. In secure data use cases, a user needs to be able to detect access controlled data as part of data discovery and evaluation. The example demonstrates how to express access control using HTTP basic authentication for a given data access service.
```
Comment (tomkralidis) [inline standard/sections/clause_7_normative_text.adoc:437]:
```suggestion
For recommended data, WNM links may also provide links to resources that implement access control in support of authentication and authorization. In secure data use cases, a user needs to be able to detect access controlled data as part of data discovery and evaluation. The example demonstrates how to express access control using HTTP Basic authentication for a given data access service.
```

**Review by tomkralidis — CHANGES_REQUESTED** [review]

File: standard/sections/clause_7_normative_text.adoc (hunk 3 above)
Comment (amilan17) [reply standard/sections/clause_7_normative_text.adoc:28]: leave in PR branch, but don't include publication doc because it is a FT amendment.

**Review by tomkralidis — APPROVED** [review]

**Pull request merged**